contracts/nexus-gateway/src/nexus.rs

```diff
@@ -45,26 +45,47 @@ fn parse_message_id(message_id: &str) -> Result<(nonempty::Vec<u8>, u64), Contra
     Ok((tx_id, index))
 }
 
-impl TryFrom<RouterMessage> for Message {
+impl TryFrom<connection_router::Message> for Message {
     type Error = Report<ContractError>;
 
-    fn try_from(msg: RouterMessage) -> Result<Self, ContractError> {
+    fn try_from(msg: connection_router::Message) -> Result<Self, ContractError> {
```

Comment:
why is this a try_from?

Reply:
good catch. It used to have to be a try_from but not anymore.